Sources/MapboxCoreNavigation/Logger+Internal.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import OSLog
+@_implementationOnly import MapboxCommon_Private.MBXLog_Internal
+
+internal typealias Log = NavigationLog
+
+/// :nodoc:
+public struct NavigationLog {
```

Comment:
I would suggest renaming this file to `Log.swift` or `NavigationLog.swift`. At the moment it looks like an extension to a `Logger` type which is not true.